parser/parser.y

```diff
@@ -1322,6 +1322,10 @@ DropTableStmt:
 		}
 	}
 
+DropTableOpt:
```

Comment:
TableOrTables is a better name.
This is not an option or ignorable token.
MySQL yacc file use table_or_tables.